data/src/main/java/com/microsoft/azure/kusto/data/format/CslUuidFormat.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.azure.kusto.data.format;
+
+import com.microsoft.azure.kusto.data.Ensure;
+
+import java.util.UUID;
+
+public class CslUuidFormat extends CslFormat {

```

Comment:
On the one hand, it seems Java has no defined convention for this scenario.
https://stackoverflow.com/questions/2236807/java-naming-convention-with-acronyms brings up a few ideas, including:
- "For 4+ letter acronyms, use mixed case. The standard library does this, and it just makes good sense."
- Several "canonical" Java classnames, showing there's clearly no consistent approach.

On the other hand we import (literally 2 lines above) a class with the acronym-in-all-caps convention :-)
I would argue it makes sense there because it's **only** the acronym.

WDYT?